deploy/docker/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM ubuntu:14.04
 
-ENV http_proxy='http://172.28.40.9:3128/'
-ENV https_proxy='http://172.28.40.9:3128/'
-ENV ftp_proxy='http://172.28.40.9:3128/'
-ENV no_proxy='127.0.0.1,localhost'
+#ENV http_proxy='http://172.28.40.9:3128/'
```

Comment:
Should we define https proxy var as well. I think it is necessary for https_proxy.